### PR TITLE
Major quill refactor

### LINKED ISSUE #33: Fix existing Docker configuration for development

### PR DESCRIPTION
**Pull request by nyaculak**
Cleaned up backend and made Docker work by removing gulp for sass compilation
Also fixes #33

### CODE REVIEW

File: app/server/routes/api.js [hunk 1]
```diff
@@ -181,34 +171,11 @@ module.exports = function(router) {
...
    */
   router.post('/users/:id/decline', isOwnerOrAdmin, function(req, res){
-    var confirmation = req.body.confirmation;
-    var id = req.params.id;
+    const id = req.params.id;
 
     UserController.declineById(id, defaultResponse(req, res));
   });
 
-  /**
```
Comment (timothychen01) [inline app/server/routes/api.js]: So we're not letting them update/reset their password?

File: package.json [hunk 2]
```diff
@@ -1,24 +1,15 @@
 {
-  "name": "hackmit-reg",
-  "description": "mean stack registration system",
+  "name": "bitcamp-quill",
```
Comment (timothychen01) [inline package.json]: Maybe we should do another PR to get rid of all `quill` references and come up with our own project codename since we're doing a pretty hefty re-write ;)
For now maybe `hackathon-reg` or `bitcamp-reg`?

File: scripts/acceptUsers.js [hunk 3]
```diff
@@ -1,20 +0,0 @@
-require('dotenv').load();
```
Comment (timothychen01) [inline scripts/acceptUsers.js]: Why are we getting rid of the four js script files?

File: app/server/routes/api.js (hunk 1 above)
Comment (nyaculak) [reply app/server/routes/api.js]: We are, but we are only doing it through email

File: package.json (hunk 2 above)
Comment (nyaculak) [reply package.json]: Agreed, we should change the name.

File: scripts/acceptUsers.js (hunk 3 above)
Comment (nyaculak) [reply scripts/acceptUsers.js]: We never really used them anyway. They aren't actually part of the application.

File: app.js [hunk 4]
```diff
@@ -12,9 +12,6 @@ var mongoose        = require('mongoose');
 var port            = process.env.PORT || 3000;
 var database        = process.env.DATABASE || process.env.MONGODB_URI || "mongodb://localhost:27017";
 
-var settingsConfig  = require('./config/settings');
-var adminConfig     = require('./config/admin');
```
[the remark this answers is not in the document]
Comment (nyaculak) [reply app.js]: This was a mistake. I'm adding them back

File: app/server/routes/api.js (hunk 1 above)
Comment (timothychen01) [reply app/server/routes/api.js]: Gotcha!

File: scripts/acceptUsers.js (hunk 3 above)
Comment (timothychen01) [reply scripts/acceptUsers.js]: Gotcha, just for testing and stuff I guess? Maybe let's keep them in just in case?
